scene/gui/code_edit.cpp

```diff
@@ -2399,6 +2399,19 @@ void CodeEdit::set_symbol_lookup_word_as_valid(bool p_valid) {
 	}
 }
 
+/* Visual */
+Color CodeEdit::_get_brace_mismatch_color() const {
+	return theme_cache.brace_mismatch_color;
+}
+
+Color CodeEdit::_get_code_folding_color() const {
+	return theme_cache.code_folding_color;
+}
+
+Ref<Texture2D> CodeEdit::_get_folded_eol_icon() const {
+	return theme_cache.folded_eol_icon;
+}
+
```

Comment:
Trying to keep private methods below `_bind_methods`

Reply:
Generally in Godot codebase we put bind methods right before the constructor. At least this is true to the majority of the GUI code.